pkg/operators/otel-metrics/otel-metrics.go

```diff
@@ -374,6 +398,44 @@ func (mc *metricsCollector) addValHistFunc(f datasource.FieldAccessor) error {
 	switch f.Type() {
 	default:
 		return fmt.Errorf("unsupported field type for metrics value %q: %s", f.Name(), f.Type())
+	case api.ArrayOf(api.Kind_Uint32):
```

Comment:
It might be obvious, but why is only the case for `uint32` different or changed?

Reply:
It's the only kind of _array_ (all other cases below are non-array) supported for the histograms currently. I should probably add the 64bit variant right away, though.